### DISCUSSION TITLE
Mention chat not show in group

### DISCUSSION THREAD

**Discussion by anggatinggalklik** [tulir/whatsmeow#676] [WhatsApp protocol Q&A]
i have implement mention chat group like this 
`pattern := `@62\d{9,12}`
		re := regexp.MustCompile(pattern)
		matches := re.FindAllString(msg.Body, -1)
		if matches != nil {
			var mentionedJID []string
			for _, match := range matches {
				mentionedJID = append(mentionedJID, types.NewJID(strings.Replace(match, "@", "", 1), types.DefaultUserServer).ToNonAD().String())
			}

			if mentionedJID != nil {
				payload.ExtendedTextMessage = &waE2E.ExtendedTextMessage{
					Text: proto.String(msg.Body),
					ContextInfo: &waE2E.ContextInfo{
						MentionedJID: mentionedJID,
					},
				}
			}
		}`

no error in log but chat isnt show in group conversation

**Comment by ghost** [comment]
Use discussion tab for protocol question.

**Comment by ghost** [comment]
> i have implement mention chat group like this `pattern := `@62\d{9,12}` re := regexp.MustCompile(pattern) matches := re.FindAllString(msg.Body, -1) if matches != nil { var mentionedJID []string for _, match := range matches { mentionedJID = append(mentionedJID, types.NewJID(strings.Replace(match, "@", "", 1), types.DefaultUserServer).ToNonAD().String()) }
> 
> ```
> 		if mentionedJID != nil {
> 			payload.ExtendedTextMessage = &waE2E.ExtendedTextMessage{
> 				Text: proto.String(msg.Body),
> 				ContextInfo: &waE2E.ContextInfo{
> 					MentionedJID: mentionedJID,
> 				},
> 			}
> 		}
> 	}`
> ```
> 
> no error in log but chat isnt show in group conversation

I have been tested your code and working as expected.

**Comment by GustavoGSA** [comment]
![image](https://github.com/user-attachments/assets/0e62e3e8-25e7-451a-9fb1-ad95afaa3c70)

There is the possibility of making this type of message, it shows that it was mentioned but does not include numbers in the message.
I need to pass @number in the message but
In large groups this gets in the way of the message.
Can anyone tell me how to do this?
